### ISSUE TITLE
Error in Plater Script 'Cast - Glowing P'

### ISSUE THREAD

**Issue by endoze** [Tercioo/Plater-Nameplates#134]
**Game and Plater version**
Plater-v537.1-Retail - DF v382 - 10.0.0

**Describe the bug**
- Steps to Reproduce (what did you do to make the bug happen):
  - Fight any dreadlord mob in Zereth Mortis that casts Carrion Swarm

- Result (what happens when you follow the steps above):
  - I get Plater errors in my chat box

- Expected (what you think the expected behavior would be when following the steps):
  - No errors should output

- When the bug started (a date, wow path release, details update, etc)?
  - Dragonflight Prepatch

- Additional information:
  - I'm not sure where in the Plater repo the built in scripts are stored or I'd make a pull request myself to fix the script.
    In the 'On Show' script, when trying to set the to/from scale values on the texture scale it uses method that were renamed.

```lua
-- In 'Cast - Glowing P' 'On Show' script
-- snip
texture.scale:SetOrder(1)
texture.scale:SetDuration(0.5)
texture.scale:SetFromScale(0.2, 1) -- this should be  texture.scale:SetScaleFrom(0.2, 1)
texture.scale:SetToScale(1, 1.5)   -- this should be  texture.scale:SetScaleTo(1, 1.5)
texture.scale:SetOrigin("right", 0, 0)
-- snip
```

**Comment by Tercioo** [comment]
Thank you for reporting and specially for providing a solution as well.
I've sent an alpha version with a updated version of the script.

------ Original Message ------
From: "Endoze" ***@***.***>
To: "Tercioo/Plater-Nameplates" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: 26-Oct-22 10:58:53 PM
Subject: [Tercioo/Plater-Nameplates] Error in Plater Script 'Cast - 
Glowing P' (Issue #134)

**Comment by endoze** [comment]
For my own sanity, is there some place these built in scripts live on github so I can contribute if I encounter additional errors?

**Comment by endoze** [comment]
I see the code changes. I'm assuming the correct procedure for updating these is to fix the script within plater/wow and export the script as a string. Then make a pull request to update the string associated with the script in `Plater_ScriptLibrary.lua`?

**Comment by cont1nuity** [comment]
Exactly. For easier review, a wago link for the update would be appreciated.

**Comment by endoze** [comment]
Thank you very much for clarifying. I appreciate the quick responses and look forward to contributing more where I can.
